packages/react-core/src/components/TextInput/TextInput.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/FormControls/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const inputTypes = [
+  'text',
+  'date',
+  'datetime-local',
+  'email',
+  'month',
+  'number',
+  'password',
+  'search',
+  'tel',
```

Comment:
"tel" is only supported by Safari 8 and newer

Reply:
should be ok to keep this here since we are newest versions only. Also most will fall back to text if not supported.